[gstreamer] Build failure

Package: gstreamer[core,plugins-bad,plugins-base,srt]:arm64-osx -> 1.22.5

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 15.0.0.15000040
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: db0473513 2023-09-27 (31 hours ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`
**Failure logs**

```
-- Using cached gstreamer-gstreamer-1.22.5.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/gstreamer-gstreamer-1.22.5.tar.gz
-- Applying patch fix-clang-cl.patch
-- Applying patch fix-clang-cl-gstreamer.patch
-- Applying patch fix-clang-cl-base.patch
-- Applying patch fix-clang-cl-good.patch
-- Applying patch fix-clang-cl-bad.patch
-- Applying patch fix-clang-cl-ugly.patch
-- Applying patch gstreamer-disable-no-unused.patch
-- Applying patch srtp_fix.patch
-- Applying patch fix-bz2-windows-debug-dependency.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/src/1.22.5-786d6f5b49.clean
-- Using bison: 3.8.2
-- Found external ninja('1.11.1').
-- Getting CMake variables for arm64-osx-dbg
-- Getting CMake variables for arm64-osx-rel
-- Configuring arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /opt/homebrew/bin/python3 /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/meson/meson.py -Dgpl=disabled -Dlibav=disabled -Dnls=disabled -Dges=disabled -Dbase=enabled -Dgst-plugins-base:gl-graphene=disabled -Dgst-plugins-base:alsa=disabled -Dgst-plugins-base:ogg=disabled -Dgst-plugins-base:opus=disabled -Dgst-plugins-base:pango=disabled -Dgst-plugins-base:vorbis=disabled -Dgst-plugins-base:x11=disabled -Dgst-plugins-base:xshm=disabled -Dgood=disabled -Dgst-plugins-good:bz2=disabled -Dgst-plugins-good:cairo=disabled -Dgst-plugins-good:flac=disabled -Dgst-plugins-good:gdk-pixbuf=disabled -Dgst-plugins-good:jpeg=disabled -Dgst-plugins-good:mpg123=disabled -Dgst-plugins-good:png=disabled -Dgst-plugins-good:speex=disabled -Dgst-plugins-good:taglib=disabled -Dgst-plugins-good:vpx=disabled -Dugly=disabled -Dgst-plugins-ugly:x264=disabled -Dbad=enabled -Dgst-plugins-bad:aes=disabled -Dgst-plugins-bad:assrender=disabled -Dgst-plugins-bad:bz2=disabled -Dgst-plugins-bad:chromaprint=disabled -Dgst-plugins-bad:closedcaption=disabled -Dgst-plugins-bad:colormanagement=disabled -Dgst-plugins-bad:dash=disabled -Dgst-plugins-bad:dc1394=disabled -Dgst-plugins-bad:dtls=disabled -Dgst-plugins-bad:faad=disabled -Dgst-plugins-bad:fdkaac=disabled -Dgst-plugins-bad:fluidsynth=disabled -Dgst-plugins-bad:libde265=disabled -Dgst-plugins-bad:microdns=disabled -Dgst-plugins-bad:modplug=disabled -Dgst-plugins-bad:openal=disabled -Dgst-plugins-bad:openh264=disabled -Dgst-plugins-bad:openjpeg=disabled -Dgst-plugins-bad:openmpt=disabled -Dgst-plugins-bad:opus=disabled -Dgst-plugins-bad:smoothstreaming=disabled -Dgst-plugins-bad:sndfile=disabled -Dgst-plugins-bad:soundtouch=disabled -Dgst-plugins-bad:srt=enabled -Dgst-plugins-bad:srtp=disabled -Dgst-plugins-bad:webp=disabled -Dgst-plugins-bad:webrtc=disabled -Dgst-plugins-bad:wildmidi=disabled -Dgst-plugins-bad:x11=disabled -Dgst-plugins-bad:x265=disabled -Dgst-plugins-bad:asio=disabled -Dpython=disabled -Dlibnice=disabled -Ddevtools=disabled -Drtsp_server=disabled -Domx=disabled -Dvaapi=disabled -Dsharp=disabled -Drs=disabled -Dgst-examples=disabled -Dtls=disabled -Dqt5=disabled -Dtests=disabled -Dexamples=disabled -Dintrospection=disabled -Dorc=disabled -Ddoc=disabled -Dgtk_doc=disabled -Dgstreamer:check=disabled -Dgstreamer:libunwind=disabled -Dgstreamer:libdw=disabled -Dgstreamer:dbghelp=disabled -Dgstreamer:bash-completion=disabled -Dgstreamer:coretracers=disabled -Dgstreamer:benchmarks=disabled -Dgstreamer:gst_debug=true -Dgst-plugins-base:gl_winsys=auto -Dgst-plugins-base:gl_platform=auto -Dgst-plugins-base:cdparanoia=disabled -Dgst-plugins-base:libvisual=disabled -Dgst-plugins-base:theora=disabled -Dgst-plugins-base:tremor=disabled -Dgst-plugins-base:xvideo=disabled -Dgst-plugins-good:aalib=disabled -Dgst-plugins-good:directsound=auto -Dgst-plugins-good:dv=disabled -Dgst-plugins-good:dv1394=disabled -Dgst-plugins-good:gtk3=disabled -Dgst-plugins-good:jack=disabled -Dgst-plugins-good:lame=disabled -Dgst-plugins-good:libcaca=disabled -Dgst-plugins-good:oss=disabled -Dgst-plugins-good:oss4=disabled -Dgst-plugins-good:osxaudio=auto -Dgst-plugins-good:osxvideo=auto -Dgst-plugins-good:pulse=auto -Dgst-plugins-good:qt5=disabled -Dgst-plugins-good:shout2=disabled -Dgst-plugins-good:soup=disabled -Dgst-plugins-good:twolame=disabled -Dgst-plugins-good:waveform=auto -Dgst-plugins-good:wavpack=disabled -Dgst-plugins-ugly:a52dec=disabled -Dgst-plugins-ugly:amrnb=disabled -Dgst-plugins-ugly:amrwbdec=disabled -Dgst-plugins-ugly:cdio=disabled -Dgst-plugins-ugly:dvdread=disabled -Dgst-plugins-ugly:mpeg2dec=disabled -Dgst-plugins-ugly:sidplay=disabled -Dgst-plugins-bad:aom=disabled -Dgst-plugins-bad:avtp=disabled -Dgst-plugins-bad:androidmedia=auto -Dgst-plugins-bad:applemedia=auto -Dgst-plugins-bad:asio-sdk-path= -Dgst-plugins-bad:bluez=disabled -Dgst-plugins-bad:bs2b=disabled -Dgst-plugins-bad:curl=disabled -Dgst-plugins-bad:curl-ssh2=disabled -Dgst-plugins-bad:d3dvideosink=auto -Dgst-plugins-bad:d3d11=auto -Dgst-plugins-bad:decklink=disabled -Dgst-plugins-bad:directfb=disabled -Dgst-plugins-bad:directsound=auto -Dgst-plugins-bad:dts=disabled -Dgst-plugins-bad:dvb=auto -Dgst-plugins-bad:faac=disabled -Dgst-plugins-bad:fbdev=auto -Dgst-plugins-bad:flite=disabled -Dgst-plugins-bad:gl=auto -Dgst-plugins-bad:gme=disabled -Dgst-plugins-bad:gs=disabled -Dgst-plugins-bad:gsm=disabled -Dgst-plugins-bad:ipcpipeline=auto -Dgst-plugins-bad:iqa=disabled -Dgst-plugins-bad:kate=disabled -Dgst-plugins-bad:kms=disabled -Dgst-plugins-bad:ladspa=disabled -Dgst-plugins-bad:ldac=disabled -Dgst-plugins-bad:lv2=disabled -Dgst-plugins-bad:mediafoundation=auto -Dgst-plugins-bad:mpeg2enc=disabled -Dgst-plugins-bad:mplex=disabled -Dgst-plugins-bad:msdk=disabled -Dgst-plugins-bad:musepack=disabled -Dgst-plugins-bad:neon=disabled -Dgst-plugins-bad:nvcodec=enabled -Dgst-plugins-bad:onnx=disabled -Dgst-plugins-bad:openaptx=disabled -Dgst-plugins-bad:opencv=disabled -Dgst-plugins-bad:openexr=disabled -Dgst-plugins-bad:openni2=disabled -Dgst-plugins-bad:opensles=disabled -Dgst-plugins-bad:qroverlay=disabled -Dgst-plugins-bad:resindvd=disabled -Dgst-plugins-bad:rsvg=disabled -Dgst-plugins-bad:rtmp=disabled -Dgst-plugins-bad:sbc=disabled -Dgst-plugins-bad:sctp=auto -Dgst-plugins-bad:shm=disabled -Dgst-plugins-bad:spandsp=disabled -Dgst-plugins-bad:svthevcenc=disabled -Dgst-plugins-bad:teletext=disabled -Dgst-plugins-bad:tinyalsa=disabled -Dgst-plugins-bad:transcode=disabled -Dgst-plugins-bad:ttml=disabled -Dgst-plugins-bad:uvch264=disabled -Dgst-plugins-bad:va=disabled -Dgst-plugins-bad:voaacenc=disabled -Dgst-plugins-bad:voamrwbenc=disabled -Dgst-plugins-bad:vulkan=auto -Dgst-plugins-bad:wasapi=auto -Dgst-plugins-bad:wasapi2=auto -Dgst-plugins-bad:wayland=auto -Dgst-plugins-bad:winks=disabled -Dgst-plugins-bad:winscreencap=auto -Dgst-plugins-bad:zbar=disabled -Dgst-plugins-bad:zxing=disabled -Dgst-plugins-bad:wpe=disabled -Dgst-plugins-bad:magicleap=disabled -Dgst-plugins-bad:v4l2codecs=disabled -Dgst-plugins-bad:isac=disabled --buildtype plain --backend ninja --wrap-mode nodownload --libdir lib --native /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/meson-arm64-osx-dbg.log -Ddebug=true --prefix /Users/leanderSchulten/git_projekte/vcpkg/packages/gstreamer_arm64-osx/debug --includedir ../include -Dcmake_prefix_path=['/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug','/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx'] /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/src/1.22.5-786d6f5b49.clean
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg
    Error code: 1
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/config-arm64-osx-dbg-meson-log.txt.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/config-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_meson.cmake:431 (vcpkg_execute_required_process)
  ports/gstreamer/portfile.cmake:146 (vcpkg_configure_meson)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/config-arm64-osx-dbg-meson-log.txt.log</summary>

```
Build started at 2023-09-28T17:44:37.698466
Main binary: /opt/homebrew/opt/python@3.11/bin/python3.11
Build Options: -Dgpl=disabled -Dlibav=disabled -Dnls=disabled -Dges=disabled -Dbase=enabled -Dgst-plugins-base:gl-graphene=disabled -Dgst-plugins-base:alsa=disabled -Dgst-plugins-base:ogg=disabled -Dgst-plugins-base:opus=disabled -Dgst-plugins-base:pango=disabled -Dgst-plugins-base:vorbis=disabled -Dgst-plugins-base:x11=disabled -Dgst-plugins-base:xshm=disabled -Dgood=disabled -Dgst-plugins-good:bz2=disabled -Dgst-plugins-good:cairo=disabled -Dgst-plugins-good:flac=disabled -Dgst-plugins-good:gdk-pixbuf=disabled -Dgst-plugins-good:jpeg=disabled -Dgst-plugins-good:mpg123=disabled -Dgst-plugins-good:png=disabled -Dgst-plugins-good:speex=disabled -Dgst-plugins-good:taglib=disabled -Dgst-plugins-good:vpx=disabled -Dugly=disabled -Dgst-plugins-ugly:x264=disabled -Dbad=enabled -Dgst-plugins-bad:aes=disabled -Dgst-plugins-bad:assrender=disabled -Dgst-plugins-bad:bz2=disabled -Dgst-plugins-bad:chromaprint=disabled -Dgst-plugins-bad:closedcaption=disabled -Dgst-plugins-bad:colormanagement=disabled -Dgst-plugins-bad:dash=disabled -Dgst-plugins-bad:dc1394=disabled -Dgst-plugins-bad:dtls=disabled -Dgst-plugins-bad:faad=disabled -Dgst-plugins-bad:fdkaac=disabled -Dgst-plugins-bad:fluidsynth=disabled -Dgst-plugins-bad:libde265=disabled -Dgst-plugins-bad:microdns=disabled -Dgst-plugins-bad:modplug=disabled -Dgst-plugins-bad:openal=disabled -Dgst-plugins-bad:openh264=disabled -Dgst-plugins-bad:openjpeg=disabled -Dgst-plugins-bad:openmpt=disabled -Dgst-plugins-bad:opus=disabled -Dgst-plugins-bad:smoothstreaming=disabled -Dgst-plugins-bad:sndfile=disabled -Dgst-plugins-bad:soundtouch=disabled -Dgst-plugins-bad:srt=enabled -Dgst-plugins-bad:srtp=disabled -Dgst-plugins-bad:webp=disabled -Dgst-plugins-bad:webrtc=disabled -Dgst-plugins-bad:wildmidi=disabled -Dgst-plugins-bad:x11=disabled -Dgst-plugins-bad:x265=disabled -Dgst-plugins-bad:asio=disabled -Dpython=disabled -Dlibnice=disabled -Ddevtools=disabled -Drtsp_server=disabled -Domx=disabled -Dvaapi=disabled -Dsharp=disabled -Drs=disabled -Dgst-examples=disabled -Dtls=disabled -Dqt5=disabled -Dtests=disabled -Dexamples=disabled -Dintrospection=disabled -Dorc=disabled -Ddoc=disabled -Dgtk_doc=disabled -Dgstreamer:check=disabled -Dgstreamer:libunwind=disabled -Dgstreamer:libdw=disabled -Dgstreamer:dbghelp=disabled -Dgstreamer:bash-completion=disabled -Dgstreamer:coretracers=disabled -Dgstreamer:benchmarks=disabled -Dgstreamer:gst_debug=true -Dgst-plugins-base:gl_winsys=auto -Dgst-plugins-base:gl_platform=auto -Dgst-plugins-base:cdparanoia=disabled -Dgst-plugins-base:libvisual=disabled -Dgst-plugins-base:theora=disabled -Dgst-plugins-base:tremor=disabled -Dgst-plugins-base:xvideo=disabled -Dgst-plugins-good:aalib=disabled -Dgst-plugins-good:directsound=auto -Dgst-plugins-good:dv=disabled -Dgst-plugins-good:dv1394=disabled -Dgst-plugins-good:gtk3=disabled -Dgst-plugins-good:jack=disabled -Dgst-plugins-good:lame=disabled
...
Skipped 5008 lines
...
Looking for framework IOSurface in /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks
Running compile:
Working directory:  /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmpn2n7fdi4
Command line:  ccache c++ /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmpn2n7fdi4/testfile.mm -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmpn2n7fdi4/output.exe -O0 -Werror=implicit-function-declaration -F/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks -framework IOSurface 

Code:
 int main(void) { return 0; }
Compiler stdout:
 
Compiler stderr:
 
Run-time dependency iosurface found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/IOSurface.framework)
Dependency appleframeworks (modules: OpenGL) found: YES unknown (cached)
Dependency bluez skipped: feature bluez disabled
Dependency gio-unix-2.0 skipped: feature bluez disabled
Running compile:
Working directory:  /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmppedtauav
Command line:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmppedtauav/testfile.c -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmppedtauav/output.obj -c -fPIC -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -g -O0 -Werror=implicit-function-declaration -Werror=unknown-warning-option -Werror=unused-command-line-argument -Werror=ignored-optimization-argument 

Code:
 
#include <linux/dvb/version.h>
#if DVB_API_VERSION < 5 || (DVB_API_VERSION == 5 && DVB_API_VERSION_MINOR <= 4)
#error "Incompatible dvb API"
#endif'

Compiler stdout:
 
Compiler stderr:
 /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmppedtauav/testfile.c:2:10: fatal error: 'linux/dvb/version.h' file not found
#include <linux/dvb/version.h>
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated.

Running compile:
Working directory:  /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmpxoxzzbi8
Command line:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmpxoxzzbi8/testfile.c -E -P -fPIC -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -g -P -O0 -Werror=implicit-function-declaration 

Code:
 
        #ifdef __has_include
         #if !__has_include("linux/fb.h")
          #error "Header 'linux/fb.h' could not be found"
         #endif
        #else
         #include <linux/fb.h>
        #endif
Compiler stdout:
 

Compiler stderr:
 /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-private/tmpxoxzzbi8/testfile.c:4:12: error: "Header 'linux/fb.h' could not be found"
          #error "Header 'linux/fb.h' could not be found"
           ^
1 error generated.

Has header "linux/fb.h" : NO 
Library ml_audio skipped: feature magicleap disabled
Library lumin_rt_core_1_7 skipped: feature magicleap disabled
Library lumin_rt_app_1_7 skipped: feature magicleap disabled

../src/1.22.5-786d6f5b49.clean/subprojects/gst-plugins-bad/sys/nvcodec/meson.build:35:4: ERROR: Problem encountered: The nvcodec was enabled explicitly, but required gstcuda dependency is not found
```
</details>
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/config-arm64-osx-dbg-out.log</summary>

```
WARNING: Recommend using either -Dbuildtype or -Doptimization + -Ddebug. Using both is redundant since they override each other. See: https://mesonbuild.com/Builtin-options.html#build-type-options
The Meson build system
Version: 0.62.1
Source dir: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/src/1.22.5-786d6f5b49.clean
Build dir: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg
Build type: native build
Project name: gstreamer-full
Project version: 1.22.5
C compiler for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc (clang 15.0.0 "Apple clang version 15.0.0 (clang-1500.0.40.1)")
C linker for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc ld64 unknown version
Compiler for language c for the build machine not found.
Host machine cpu family: aarch64
Host machine cpu: arm64
Program python found: YES (/opt/homebrew/bin/python3)

Executing subproject macos-bison-binary 

macos-bison-binary| Project name: win-flex-bison-binary
macos-bison-binary| Project version: 3.7.6
macos-bison-binary| Message: Downloading and extracting bison for macOS x64...
macos-bison-binary| Configuring bison.py using configuration
macos-bison-binary| Program bison.py found: YES (/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/subprojects/macos-bison-binary/bison.py)
macos-bison-binary| Build targets in project: 0
macos-bison-binary| Subproject macos-bison-binary finished.


Executing subproject gstreamer 

gstreamer| Project name: gstreamer
gstreamer| Project version: 1.22.5
gstreamer| C compiler for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc (clang 15.0.0 "Apple clang version 15.0.0 (clang-1500.0.40.1)")
gstreamer| C linker for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc ld64 unknown version
gstreamer| Checking if "building for iOS" compiles: NO
gstreamer| Objective-C compiler for the host machine: ccache cc (clang 15.0.0)
gstreamer| Objective-C linker for the host machine: cc ld64 unknown version
gstreamer| Compiler for C supports link arguments -Wl,-Bsymbolic-functions: NO
gstreamer| Compiler for C supports link arguments -Wl,-z,nodelete: NO
gstreamer| Compiler for C supports arguments -fvisibility=hidden: YES
gstreamer| Compiler for C supports arguments -fno-strict-aliasing: YES
gstreamer| Message: Disabling GLib cast checks
gstreamer| Has header "dlfcn.h" : YES
gstreamer| Has header "inttypes.h" : YES
gstreamer| Has header "memory.h" : YES
gstreamer| Has header "poll.h" : YES
gstreamer| Has header "stdint.h" : YES
gstreamer| Has header "stdio_ext.h" : NO
gstreamer| Has header "strings.h" : YES
gstreamer| Has header "string.h" : YES
gstreamer| Has header "sys/param.h" : YES
gstreamer| Has header "sys/poll.h" : YES
gstreamer| Has header "sys/prctl.h" : NO
gstreamer| Has header "sys/socket.h" : YES
gstreamer| Has header "sys/stat.h" : YES
gstreamer| Has header "sys/times.h" : YES
gstreamer| Has header "sys/time.h" : YES
gstreamer| Has header "sys/types.h" : YES
gstreamer| Has header "sys/utsname.h" : YES
gstreamer| Has header "sys/wait.h" : YES
gstreamer| Has header "ucontext.h" : YES
gstreamer| Has header "unistd.h" : YES
gstreamer| Has header "sys/resource.h" : YES
gstreamer| Has header "sys/uio.h" : YES
gstreamer| Checking whether type "struct tm" has member "tm_gmtoff" : YES
gstreamer| Checking for function "gmtime_r" : YES
gstreamer| Checking for function "sigaction" : YES
gstreamer| Checking for function "getrusage" : YES
gstreamer| Checking for function "fseeko" : YES
gstreamer| Checking for function "ftello" : YES
gstreamer| Checking for function "poll" : YES
gstreamer| Checking for function "ppoll" : NO
gstreamer| Checking for function "pselect" : YES
gstreamer| Checking for function "getpagesize" : YES
gstreamer| Checking for function "clock_gettime" : YES
gstreamer| Checking for function "clock_nanosleep" : NO
gstreamer| Checking for function "strnlen" : YES
gstreamer| Checking for function "getline" : YES
gstreamer| Checking for function "mkstemp" : YES
gstreamer| Checking for function "alarm" : YES
gstreamer| Checking for function "gettimeofday" : YES
gstreamer| Checking for function "localtime_r" : YES
gstreamer| Checking if "pthread_setname_np(const char*)" : links: YES
gstreamer| Header <pthread.h> has symbol "pthread_condattr_setclock" : NO
gstreamer| Header <pthread.h> has symbol "pthread_cond_timedwait_relative_np" : YES
gstreamer| Checking if "futex(2) system call" compiles: NO
gstreamer| Checking if "futex(2) system call" compiles: NO
gstreamer| Checking if "posix timers from time.h" compiles: NO
gstreamer| Checking if "monotonic clock from time.h" compiles: NO
gstreamer| Checking if "__uint128_t available" compiles: YES
gstreamer| Checking for function "getpid" : YES
gstreamer| Checking for function "strdup" : YES
gstreamer| Checking for function "strsignal" : YES
gstreamer| Checking for type "clockid_t" : YES
gstreamer| Checking for type "timer_t" : NO
gstreamer| Checking whether type "struct timespec" has members "tv_sec", "tv_nsec" : YES
gstreamer| Checking whether type "struct itimerspec" has members "it_interval", "it_value" : NO
gstreamer| Dependency libunwind skipped: feature libunwind disabled
gstreamer| Dependency libdw skipped: feature libdw disabled
gstreamer| Has header "dbghelp.h" skipped: feature dbghelp disabled
gstreamer| Checking for function "backtrace" : YES
gstreamer| Message: Minimal support for stack traces, no source info.
gstreamer| Has header "execinfo.h" : YES
gstreamer| Checking for function "backtrace" : YES
gstreamer| Compiler for C supports arguments -Wmissing-declarations: YES
gstreamer| Compiler for C supports arguments -Wmissing-prototypes: YES
gstreamer| Compiler for C supports arguments -Wredundant-decls: YES
gstreamer| Compiler for C supports arguments -Wundef: YES
gstreamer| Compiler for C supports arguments -Wwrite-strings: YES
gstreamer| Compiler for C supports arguments -Wformat: YES
gstreamer| Compiler for C supports arguments -Wformat-nonliteral: YES
gstreamer| Compiler for C supports arguments -Wformat-security: YES
gstreamer| Compiler for C supports arguments -Wold-style-definition: YES
gstreamer| Compiler for C supports arguments -Winit-self: YES
gstreamer| Compiler for C supports arguments -Wmissing-include-dirs: YES
gstreamer| Compiler for C supports arguments -Waddress: YES
gstreamer| Compiler for C supports arguments -Waggregate-return: YES
gstreamer| Compiler for C supports arguments -Wno-multichar: YES
gstreamer| Compiler for C supports arguments -Wvla: YES
gstreamer| Compiler for C supports arguments -Wpointer-arith: YES
gstreamer| Library gmp found: NO
gstreamer| Library gsl found: NO
gstreamer| Library gslcblas found: NO
gstreamer| Library dl found: YES
gstreamer| Checking for function "dladdr" with dependency -ldl: YES
gstreamer| Found pkg-config: /opt/homebrew/bin/pkg-config (0.29.2)
gstreamer| Run-time dependency glib-2.0 found: YES 2.76.5
gstreamer| Run-time dependency gobject-2.0 found: YES 2.76.5
gstreamer| Run-time dependency gmodule-no-export-2.0 found: YES 2.76.5
gstreamer| Run-time dependency gio-2.0 found: YES 2.76.5
gstreamer| Run-time dependency gio-unix-2.0 found: YES 2.76.5
gstreamer| Library m found: YES
gstreamer| Library rt found: NO
gstreamer| Program g-ir-scanner skipped: feature introspection disabled
gstreamer| Program python found: YES (/opt/homebrew/bin/python3)
gstreamer| Dependency bash-completion skipped: feature bash-completion disabled
gstreamer| Run-time dependency appleframeworks found: YES (Cocoa)
gstreamer| Configuring gstconfig.h using configuration
gstreamer| Configuring gstversion.h using configuration
gstreamer| Program glib-mkenums found: YES
gstreamer| Program glib-mkenums found: YES
gstreamer| Program flex found: YES
gstreamer| Program get_flex_version.py found: YES (/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/src/1.22.5-786d6f5b49.clean/subprojects/gstreamer/gst/parse/get_flex_version.py)
gstreamer| Message: flex version 2.6.4 >= 2.5.31: YES
gstreamer| Configuring gen_lex.py using configuration
gstreamer| Program bison.py found: YES (overridden)
gstreamer| Message: bison version 3.7.6 >= 2.4: YES
gstreamer| Configuring grammar.y using configuration
gstreamer| Configuring gen_grammar.py using configuration
gstreamer| Compiler for C supports arguments -Wno-format-nonliteral: YES
gstreamer| Checking if "long long" compiles: YES
gstreamer| Checking if "uintmax_t in inttypes.h" compiles: YES
gstreamer| Checking if "uintmax_t in stdint.h" compiles: YES
gstreamer| Program glib-mkenums found: YES
gstreamer| Program glib-mkenums found: YES
gstreamer| Message: PTP supported on darwin.
gstreamer| Checking if "SIOCGIFCONF, SIOCGIFFLAGS and SIOCGIFHWADDR available" compiles: NO
gstreamer| Checking if "getifaddrs() and AF_LINK available" compiles: YES
gstreamer| Program setcap /usr/sbin/setcap /sbin/setcap found: NO
gstreamer| Found CMake: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.27.1-osx/cmake-3.27.1-macos-universal/CMake.app/Contents/bin/cmake (3.27.1)
gstreamer| Run-time dependency libcap found: NO (tried pkgconfig, framework and cmake)
gstreamer| Message: How to install gst-ptp-helper: setuid-root
gstreamer| Configuring libgstreamer-1.0.so.0.2205.0-gdb.py using configuration
gstreamer| Message: Building statically, can't build the documentation
gstreamer| Program xgettext skipped: feature nls disabled
gstreamer| Program scripts/extract-release-date-from-doap-file.py found: YES (/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/src/1.22.5-786d6f5b49.clean/subprojects/gstreamer/scripts/extract-release-date-from-doap-file.py)
gstreamer| Message: Package release date: 2023-07-20
gstreamer| Configuring config.h using configuration
gstreamer| Build targets in project: 19
gstreamer| Subproject gstreamer finished.


Executing subproject gst-plugins-base 

gst-plugins-base| Project name: gst-plugins-base
gst-plugins-base| Project version: 1.22.5
gst-plugins-base| C compiler for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc (clang 15.0.0 "Apple clang version 15.0.0 (clang-1500.0.40.1)")
gst-plugins-base| C linker for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc ld64 unknown version
gst-plugins-base| C++ compiler for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ (clang 15.0.0 "Apple clang version 15.0.0 (clang-1500.0.40.1)")
gst-plugins-base| C++ linker for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ ld64 unknown version
gst-plugins-base| Objective-C compiler for the host machine: ccache cc (clang 15.0.0)
gst-plugins-base| Objective-C linker for the host machine: cc ld64 unknown version
gst-plugins-base| Compiler for C supports link arguments -Wl,-Bsymbolic-functions: NO (cached)
gst-plugins-base| Compiler for C supports link arguments -Wl,-z,nodelete: NO (cached)
gst-plugins-base| Compiler for C supports arguments -fvisibility=hidden: YES (cached)
gst-plugins-base| Compiler for C supports arguments -fno-strict-aliasing: YES (cached)
gst-plugins-base| Message: Disabling GLib cast checks
gst-plugins-base| Has header "dlfcn.h" : YES (cached)
gst-plugins-base| Has header "emmintrin.h" : YES
gst-plugins-base| Has header "inttypes.h" : YES (cached)
gst-plugins-base| Has header "memory.h" : YES (cached)
gst-plugins-base| Has header "netinet/in.h" : YES
gst-plugins-base| Has header "netinet/tcp.h" : YES
gst-plugins-base| Has header "process.h" : NO
gst-plugins-base| Has header "smmintrin.h" : YES
gst-plugins-base| Has header "stdint.h" : YES (cached)
gst-plugins-base| Has header "strings.h" : YES (cached)
gst-plugins-base| Has header "string.h" : YES (cached)
gst-plugins-base| Has header "sys/socket.h" : YES (cached)
gst-plugins-base| Has header "sys/stat.h" : YES (cached)
gst-plugins-base| Has header "sys/types.h" : YES (cached)
gst-plugins-base| Has header "sys/wait.h" : YES (cached)
gst-plugins-base| Has header "unistd.h" : YES (cached)
gst-plugins-base| Has header "winsock2.h" : NO
gst-plugins-base| Has header "xmmintrin.h" : YES
gst-plugins-base| Has header "linux/dma-buf.h" : NO
gst-plugins-base| Library m found: YES
gst-plugins-base| Checking for function "dcgettext" with dependency -lm: NO
gst-plugins-base| Checking for function "gmtime_r" with dependency -lm: YES
gst-plugins-base| Checking for function "localtime_r" with dependency -lm: YES
gst-plugins-base| Checking for function "lrintf" with dependency -lm: YES
gst-plugins-base| Checking for function "mmap" with dependency -lm: YES
gst-plugins-base| Checking for function "log2" with dependency -lm: YES
gst-plugins-base| Checking for size of "char" : 1
gst-plugins-base| Checking for size of "int" : 4
gst-plugins-base| Checking for size of "long" : 8
gst-plugins-base| Checking for size of "short" : 2
gst-plugins-base| Checking for size of "void*" : 8
gst-plugins-base| Compiler for C++ supports arguments -Waggregate-return: YES
gst-plugins-base| Compiler for C supports arguments -Wmissing-declarations: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wmissing-declarations: YES
gst-plugins-base| Compiler for C supports arguments -Wredundant-decls: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wredundant-decls: YES
gst-plugins-base| Compiler for C supports arguments -Wundef: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wundef: YES
gst-plugins-base| Compiler for C supports arguments -Wwrite-strings: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wwrite-strings: YES
gst-plugins-base| Compiler for C supports arguments -Wformat: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wformat: YES
gst-plugins-base| Compiler for C supports arguments -Wformat-nonliteral: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wformat-nonliteral: YES
gst-plugins-base| Compiler for C supports arguments -Wformat-security: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wformat-security: YES
gst-plugins-base| Compiler for C supports arguments -Winit-self: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Winit-self: YES
gst-plugins-base| Compiler for C supports arguments -Wmissing-include-dirs: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wmissing-include-dirs: YES
gst-plugins-base| Compiler for C supports arguments -Waddress: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Waddress: YES
gst-plugins-base| Compiler for C supports arguments -Wno-multichar: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wno-multichar: YES
gst-plugins-base| Compiler for C supports arguments -Wvla: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wvla: YES
gst-plugins-base| Compiler for C supports arguments -Wpointer-arith: YES (cached)
gst-plugins-base| Compiler for C++ supports arguments -Wpointer-arith: YES
gst-plugins-base| Compiler for C supports arguments -Wmissing-prototypes: YES (cached)
gst-plugins-base| Dependency x11 skipped: feature x11 disabled
gst-plugins-base| Dependency gio-2.0 found: YES 2.76.5 (cached)
gst-plugins-base| Dependency gio-unix-2.0 found: YES 2.76.5 (cached)
gst-plugins-base| Dependency gmodule-no-export-2.0 found: YES 2.76.5 (cached)
gst-plugins-base| Dependency gdk-pixbuf-2.0 skipped: feature examples disabled
gst-plugins-base| Dependency gtk+-3.0 skipped: feature examples disabled
gst-plugins-base| Dependency gtk+-quartz-3.0 skipped: feature examples disabled
gst-plugins-base| Dependency gstreamer-1.0 found: YES 1.22.5 (overridden)
gst-plugins-base| Dependency gstreamer-base-1.0 found: YES 1.22.5 (overridden)
gst-plugins-base| Dependency gstreamer-net-1.0 found: YES 1.22.5 (overridden)
gst-plugins-base| Dependency gstreamer-check-1.0 skipped: feature tests disabled
gst-plugins-base| Dependency gstreamer-controller-1.0 found: YES 1.22.5 (overridden)
gst-plugins-base| Dependency orc-0.4 skipped: feature orc disabled
gst-plugins-base| Program orcc skipped: feature orc disabled
gst-plugins-base| Message: Orc Compiler not found or disabled, will use backup C code
gst-plugins-base| Compiler for C supports arguments -msse: NO
gst-plugins-base| Compiler for C supports arguments -msse2: NO
gst-plugins-base| Compiler for C supports arguments -msse4.1: NO
gst-plugins-base| Message: GStreamer debug system is enabled
gst-plugins-base| Checking whether type "struct tcp_info" has member "__tcpi_reordering" : NO
gst-plugins-base| Checking whether type "struct tcp_info" has member "tcpi_reordering" : NO
gst-plugins-base| Program g-ir-scanner skipped: feature introspection disabled
gst-plugins-base| Program scripts/meson-pkg-config-file-fixup.py found: YES (/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/src/1.22.5-786d6f5b49.clean/subprojects/gst-plugins-base/scripts/meson-pkg-config-file-fixup.py)
gst-plugins-base| Program python found: YES (/opt/homebrew/bin/python3)
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Run-time dependency iso-codes found: NO (tried pkgconfig, framework and cmake)
gst-plugins-base| Run-time dependency zlib found: YES 1.3
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Configuring gstpluginsbaseversion.h using configuration
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Looking for a fallback subproject for the dependency (anonymous)
gst-plugins-base| Automatic wrap-based subproject downloading is disabled
gst-plugins-base| Subproject  gl-headers is buildable: NO (disabling)
gst-plugins-base| Dependency (anonymous) from subproject gl-headers found: NO (subproject failed to configure)
gst-plugins-base| Run-time dependency opengl found: NO (tried pkgconfig)
gst-plugins-base| Run-time dependency appleframeworks found: YES (OpenGL)
gst-plugins-base| Run-time dependency glx found: NO (tried pkgconfig)
gst-plugins-base| Library GL found: NO
gst-plugins-base| Run-time dependency bcm_host found: NO (tried pkgconfig, framework and cmake)
gst-plugins-base| Library bcm_host found: NO
gst-plugins-base| Run-time dependency glesv2 found: NO (tried pkgconfig, framework and cmake)
gst-plugins-base| Library GLESv2 found: NO
gst-plugins-base| Has header "GLES2/gl2.h" : YES
gst-plugins-base| Run-time dependency egl found: NO (tried pkgconfig, framework and cmake)
gst-plugins-base| Library EGL found: NO
gst-plugins-base| Message: Could not find EGL libraries for wayland
gst-plugins-base| Run-time dependency x11-xcb found: YES 1.8.6
gst-plugins-base| Checking for type "EGL_DISPMANX_WINDOW_T" with dependencies appleframeworks, -lEGL, -lbcm_host: NO
gst-plugins-base| Compiler for Objective-C supports arguments -fobjc-arc: YES
gst-plugins-base| Run-time dependency appleframeworks found: YES (Foundation)
gst-plugins-base| Run-time dependency appleframeworks found: YES (QuartzCore)
gst-plugins-base| Run-time dependency appleframeworks found: YES (CoreFoundation)
gst-plugins-base| Dependency appleframeworks (modules: Cocoa) found: YES unknown (cached)
gst-plugins-base| Run-time dependency gudev-1.0 found: NO (tried pkgconfig, framework and cmake)
gst-plugins-base| Run-time dependency libdrm found: NO (tried pkgconfig, framework and cmake)
gst-plugins-base| Run-time dependency gbm found: NO (tried pkgconfig, framework and cmake)
gst-plugins-base| Checking for function "glTexDirectVIV" with dependency -lGLESv2: NO
gst-plugins-base| Checking for type "GLeglImageOES" with dependency appleframeworks: NO
gst-plugins-base| Checking for type "GLchar" with dependency appleframeworks: YES
gst-plugins-base| Checking for type "GLsizeiptr" with dependency appleframeworks: YES
gst-plugins-base| Checking for type "GLintptr" with dependency appleframeworks: YES
gst-plugins-base| Checking for type "GLsync" with dependency appleframeworks: YES
gst-plugins-base| Checking for type "GLuint64" with dependency appleframeworks: YES
gst-plugins-base| Checking for type "GLint64" with dependency appleframeworks: YES
gst-plugins-base| Configuring gstglconfig.h using configuration
gst-plugins-base| Program glib-mkenums found: YES
gst-plugins-base| Program gl_mkenum.py found: YES (/opt/homebrew/opt/python@3.11/bin/python3.11 /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/src/1.22.5-786d6f5b49.clean/subprojects/gst-plugins-base/gst-libs/gst/gl/gl_mkenum.py)
gst-plugins-base| Dependency alsa skipped: feature alsa disabled
gst-plugins-base| Dependency graphene-gobject-1.0 skipped: feature gl-graphene disabled
gst-plugins-base| Run-time dependency libpng found: YES 1.6.40
gst-plugins-base| Run-time dependency libjpeg found: YES 3.0.0
gst-plugins-base| Dependency appleframeworks (modules: CoreFoundation) found: YES unknown (cached)
gst-plugins-base| Dependency appleframeworks (modules: Foundation) found: YES unknown (cached)
gst-plugins-base| Dependency appleframeworks (modules: QuartzCore) found: YES unknown (cached)
gst-plugins-base| Compiler for Objective-C supports arguments -fobjc-arc: YES (cached)
gst-plugins-base| Dependency libvisual-0.4 skipped: feature libvisual disabled
gst-plugins-base| Dependency ogg skipped: feature ogg disabled
gst-plugins-base| Dependency opus skipped: feature opus disabled
gst-plugins-base| Dependency pangocairo skipped: feature pango disabled
gst-plugins-base| Dependency theoradec skipped: feature theora disabled
gst-plugins-base| Dependency theoraenc skipped: feature theora disabled
gst-plugins-base| Dependency vorbis skipped: feature vorbis disabled
gst-plugins-base| Dependency vorbisenc skipped: feature vorbis disabled
gst-plugins-base| Dependency vorbisidec skipped: feature tremor disabled
gst-plugins-base| Program xgettext skipped: feature nls disabled
gst-plugins-base| Message: Building statically, can't build the documentation
gst-plugins-base| Program scripts/extract-release-date-from-doap-file.py found: YES (/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/src/1.22.5-786d6f5b49.clean/subprojects/gst-plugins-base/scripts/extract-release-date-from-doap-file.py)
gst-plugins-base| Message: Package release date: 2023-07-20
gst-plugins-base| Configuring config.h using configuration
gst-plugins-base| Build targets in project: 73
gst-plugins-base| Subproject gst-plugins-base finished.

Subproject gst-plugins-good : skipped: feature good disabled
Subproject libnice : skipped: feature libnice disabled

Executing subproject gst-plugins-bad 

gst-plugins-bad| Project name: gst-plugins-bad
gst-plugins-bad| Project version: 1.22.5
gst-plugins-bad| C compiler for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc (clang 15.0.0 "Apple clang version 15.0.0 (clang-1500.0.40.1)")
gst-plugins-bad| C linker for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc ld64 unknown version
gst-plugins-bad| C++ compiler for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ (clang 15.0.0 "Apple clang version 15.0.0 (clang-1500.0.40.1)")
gst-plugins-bad| C++ linker for the host machine: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ ld64 unknown version
gst-plugins-bad| Compiler for language cpp for the build machine not found.
gst-plugins-bad| Objective-C compiler for the host machine: ccache cc (clang 15.0.0)
gst-plugins-bad| Objective-C linker for the host machine: cc ld64 unknown version
gst-plugins-bad| Objective-C++ compiler for the host machine: ccache c++ (clang 15.0.0)
gst-plugins-bad| Objective-C++ linker for the host machine: c++ ld64 unknown version
gst-plugins-bad| Compiler for C++ supports arguments -Wno-non-virtual-dtor: YES
gst-plugins-bad| Compiler for C supports link arguments -Wl,-Bsymbolic-functions: NO (cached)
gst-plugins-bad| Compiler for C supports link arguments -Wl,-z,nodelete: NO (cached)
gst-plugins-bad| Compiler for C++ supports link arguments -Wl,-z,nodelete: NO
gst-plugins-bad| Compiler for C supports arguments -fvisibility=hidden: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -fno-strict-aliasing: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -fno-strict-aliasing: YES
gst-plugins-bad| Message: Disabling GLib cast checks
gst-plugins-bad| Has header "dlfcn.h" : YES (cached)
gst-plugins-bad| Has header "fcntl.h" : YES
gst-plugins-bad| Has header "inttypes.h" : YES (cached)
gst-plugins-bad| Has header "memory.h" : YES (cached)
gst-plugins-bad| Has header "netinet/in.h" : YES (cached)
gst-plugins-bad| Has header "netinet/ip.h" : YES
gst-plugins-bad| Has header "netinet/tcp.h" : YES (cached)
gst-plugins-bad| Has header "pthread.h" : YES
gst-plugins-bad| Has header "stdint.h" : YES (cached)
gst-plugins-bad| Has header "stdlib.h" : YES
gst-plugins-bad| Has header "strings.h" : YES (cached)
gst-plugins-bad| Has header "string.h" : YES (cached)
gst-plugins-bad| Has header "sys/param.h" : YES (cached)
gst-plugins-bad| Has header "sys/socket.h" : YES (cached)
gst-plugins-bad| Has header "sys/stat.h" : YES (cached)
gst-plugins-bad| Has header "sys/time.h" : YES (cached)
gst-plugins-bad| Has header "sys/types.h" : YES (cached)
gst-plugins-bad| Has header "sys/utsname.h" : YES (cached)
gst-plugins-bad| Has header "unistd.h" : YES (cached)
gst-plugins-bad| Has header "windows.h" : NO
gst-plugins-bad| Has header "winsock2.h" : NO (cached)
gst-plugins-bad| Has header "ws2tcpip.h" : NO
gst-plugins-bad| Checking for function "dcgettext" : NO
gst-plugins-bad| Checking for function "getpagesize" : YES (cached)
gst-plugins-bad| Checking for function "gmtime_r" : YES (cached)
gst-plugins-bad| Checking for function "memfd_create" : NO
gst-plugins-bad| Checking for function "mmap" : YES
gst-plugins-bad| Checking for function "pipe2" : NO
gst-plugins-bad| Checking for function "getrusage" : YES
gst-plugins-bad| Checking for size of "char" : 1
gst-plugins-bad| Checking for size of "int" : 4
gst-plugins-bad| Checking for size of "long" : 8
gst-plugins-bad| Checking for size of "short" : 2
gst-plugins-bad| Checking for size of "void*" : 8
gst-plugins-bad| Compiler for C supports arguments -Wmissing-prototypes: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wold-style-definition: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wformat-nonliteral: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wmissing-declarations: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wmissing-declarations: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wredundant-decls: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wredundant-decls: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wwrite-strings: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wwrite-strings: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wformat: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wformat: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wformat-security: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wformat-security: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Winit-self: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Winit-self: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wmissing-include-dirs: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wmissing-include-dirs: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Waddress: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Waddress: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wno-multichar: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wno-multichar: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wvla: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wvla: YES (cached)
gst-plugins-bad| Compiler for C supports arguments -Wpointer-arith: YES (cached)
gst-plugins-bad| Compiler for C++ supports arguments -Wpointer-arith: YES (cached)
gst-plugins-bad| Dependency gstreamer-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-base-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-net-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-controller-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-pbutils-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-allocators-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-app-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-audio-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-fft-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-riff-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-rtp-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-rtsp-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-sdp-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-tag-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-video-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-check-1.0 skipped: feature tests disabled
gst-plugins-bad| Dependency gstreamer-gl-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Dependency gstreamer-gl-prototypes-1.0 found: YES 1.22.5 (overridden)
gst-plugins-bad| Message: GStreamer OpenGL window systems: cocoa
gst-plugins-bad| Message: GStreamer OpenGL platforms: cgl
gst-plugins-bad| Message: GStreamer OpenGL apis: gl
gst-plugins-bad| Library m found: YES
gst-plugins-bad| Dependency gio-2.0 found: YES 2.76.5 (cached)
gst-plugins-bad| Dependency gmodule-no-export-2.0 found: YES 2.76.5 (cached)
gst-plugins-bad| Dependency x11 skipped: feature x11 disabled
gst-plugins-bad| Run-time dependency avfoundation found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/AVFoundation.framework)
gst-plugins-bad| Run-time dependency videotoolbox found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/VideoToolbox.framework)
gst-plugins-bad| Dependency orc-0.4 skipped: feature orc disabled
gst-plugins-bad| Program orcc skipped: feature orc disabled
gst-plugins-bad| Message: Orc Compiler not found or disabled, will use backup C code
gst-plugins-bad| Message: GStreamer debug system is enabled
gst-plugins-bad| Program python found: YES (/opt/homebrew/bin/python3)
gst-plugins-bad| Program g-ir-scanner skipped: feature introspection disabled
gst-plugins-bad| Program glib-mkenums found: YES
gst-plugins-bad| Program glib-mkenums found: YES
gst-plugins-bad| Program glib-mkenums found: YES
gst-plugins-bad| Program glib-mkenums found: YES
gst-plugins-bad| Program glib-mkenums found: YES
gst-plugins-bad| Program glib-mkenums found: YES
gst-plugins-bad| Library MoltenVK found: NO
gst-plugins-bad| Has header "vulkan/vulkan_core.h" : NO
gst-plugins-bad| Dependency xcb skipped: feature x11 disabled
gst-plugins-bad| Dependency xkbcommon skipped: feature x11 disabled
gst-plugins-bad| Dependency xkbcommon-x11 skipped: feature x11 disabled
gst-plugins-bad| Run-time dependency wayland-client found: NO (tried pkgconfig, framework and cmake)
gst-plugins-bad| Compiler for Objective-C supports arguments -fobjc-arc: YES (cached)
gst-plugins-bad| Dependency appleframeworks (modules: Foundation) found: YES unknown (cached)
gst-plugins-bad| Dependency appleframeworks (modules: QuartzCore) found: YES unknown (cached)
gst-plugins-bad| Dependency appleframeworks (modules: CoreFoundation) found: YES unknown (cached)
gst-plugins-bad| Dependency appleframeworks (modules: Cocoa) found: YES unknown (cached)
gst-plugins-bad| Has header "vulkan/vulkan_macos.h" with dependency -lMoltenVK: NO
gst-plugins-bad| Message: No Windowing system found. vulkansink will not work
gst-plugins-bad| Program glslc found: NO
gst-plugins-bad| Run-time dependency wayland-client found: NO (tried pkgconfig, framework and cmake)
gst-plugins-bad| Run-time dependency libdrm found: NO (tried pkgconfig, framework and cmake)
gst-plugins-bad| Run-time dependency wayland-protocols found: NO (tried pkgconfig, framework and cmake)
gst-plugins-bad| Program wayland-scanner found: NO
gst-plugins-bad| Program glib-mkenums found: YES
gst-plugins-bad| Program glib-mkenums found: YES
gst-plugins-bad| Dependency nice skipped: feature webrtc disabled
gst-plugins-bad| Compiler for C supports arguments -Wno-unknown-pragmas: YES
gst-plugins-bad| Run-time dependency ltc found: NO (tried pkgconfig, framework and cmake)
gst-plugins-bad| Compiler for Objective-C supports arguments -fobjc-arc: YES (cached)
gst-plugins-bad| Run-time dependency corefoundation found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/CoreFoundation.framework)
gst-plugins-bad| Run-time dependency coremedia found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/CoreMedia.framework)
gst-plugins-bad| Run-time dependency corevideo found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/CoreVideo.framework)
gst-plugins-bad| Run-time dependency audiotoolbox found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/AudioToolbox.framework)
gst-plugins-bad| Dependency AVFoundation found: YES unknown (cached)
gst-plugins-bad| Dependency VideoToolbox found: YES unknown (cached)
gst-plugins-bad| Run-time dependency cocoa found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Cocoa.framework)
gst-plugins-bad| Run-time dependency iosurface found: YES (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/IOSurface.framework)
gst-plugins-bad| Dependency appleframeworks (modules: OpenGL) found: YES unknown (cached)
gst-plugins-bad| Dependency bluez skipped: feature bluez disabled
gst-plugins-bad| Dependency gio-unix-2.0 skipped: feature bluez disabled
gst-plugins-bad| Has header "linux/fb.h" : NO
gst-plugins-bad| Library ml_audio skipped: feature magicleap disabled
gst-plugins-bad| Library lumin_rt_core_1_7 skipped: feature magicleap disabled
gst-plugins-bad| Library lumin_rt_app_1_7 skipped: feature magicleap disabled

../src/1.22.5-786d6f5b49.clean/subprojects/gst-plugins-bad/sys/nvcodec/meson.build:35:4: ERROR: Problem encountered: The nvcodec was enabled explicitly, but required gstcuda dependency is not found

A full log can be found at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/gstreamer/arm64-osx-dbg/meson-logs/meson-log.txt
```
</details>**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "gstreamer",
      "default-features": false,
      "features": [
        "srt"
      ]
    }
  ],
  "$overrides": [
    {
      "name": "llvm",
      "version": "14.0.6#1"
    }
  ],
  "$builtin-baseline": "e68bd1cc40a13fd47d999ecff333e8b809b2c720"
}

```
</details>

--- a/ports/gstreamer/portfile.cmake
+++ b/ports/gstreamer/portfile.cmake
@@ -248,7 +248,7 @@ vcpkg_configure_meson(
         -Dgst-plugins-bad:msdk=disabled
         -Dgst-plugins-bad:musepack=disabled
         -Dgst-plugins-bad:neon=disabled
-        -Dgst-plugins-bad:nvcodec=enabled
+        -Dgst-plugins-bad:nvcodec=disabled
         -Dgst-plugins-bad:onnx=disabled # libonnxruntime not found
         -Dgst-plugins-bad:openaptx=disabled
         -Dgst-plugins-bad:opencv=disabled # opencv not found

--- a/ports/gstreamer/vcpkg.json
+++ b/ports/gstreamer/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "gstreamer",
   "version": "1.22.5",
+  "port-version": 1,
   "description": "GStreamer open-source multimedia framework core library",
   "homepage": "https://gstreamer.freedesktop.org/",
   "license": "LGPL-2.0-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3062,7 +3062,7 @@
     },
     "gstreamer": {
       "baseline": "1.22.5",
-      "port-version": 0
+      "port-version": 1
     },
     "gtest": {
       "baseline": "1.14.0",

--- a/versions/g-/gstreamer.json
+++ b/versions/g-/gstreamer.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "7f37f12f5a39fa6eea841721c51bb591039c3251",
+      "version": "1.22.5",
+      "port-version": 1
+    },
+    {
       "git-tree": "76183f0a0e77acbf09ac1ae73b1b2a27f986fcf6",
       "version": "1.22.5",
       "port-version": 0

